tests/test_main.py

```diff
@@ -39,6 +39,7 @@ def test_car_wash_station(cars, wash_station, total_cost):
     income = wash_station.serve_cars(cars)
     assert income == total_cost, f"Income should equal to {total_cost}"
 
+
```

Comment:
rollback changes in test.py

Reply:
not fixed

Reply:
Sorry, should I try to upgrade for loop again?